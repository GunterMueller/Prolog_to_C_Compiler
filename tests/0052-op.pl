main :-
	op(1000, xfx, ';'),
	op(1, xf, ['#', '~']),
	current_op(X, Y, Z), display([X, Y, Z]), nl, fail.
main.
