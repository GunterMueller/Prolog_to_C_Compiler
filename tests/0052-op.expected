[1,xf,~]
[1,xf,#]
[1000,xfx,;]
[700,xfx,@=<]
[700,xfx,@<]
[1000,xfy,,]
[700,xfx,is]
[200,xfx,**]
[700,xfx,@>]
[400,yfx,<<]
[700,xfx,@>=]
[700,xfx,\==]
[200,xfy,^]
[200,fy,+]
[500,yfx,+]
[400,yfx,>>]
[700,xfx,==]
[200,fy,-]
[500,yfx,-]
[1150,fx,initialization]
[1150,fx,public]
[1150,fx,meta_predicate]
[1150,fx,discontiguous]
[1150,fx,determinate]
[1150,fx,mode]
[900,fy,\+]
[700,xfx,=:=]
[700,xfx,=\=]
[500,yfx,\/]
[1200,fx,:-]
[1200,xfx,:-]
[700,xfx,\=]
[200,fy,\]
[500,yfx,/\]
[400,yfx,xor]
[700,xfx,=<]
[700,xfx,<]
[700,xfx,=..]
[700,xfx,=]
[400,yfx,//]
[400,yfx,/]
[1050,xfy,->]
[700,xfx,>]
[400,yfx,*]
[700,xfx,>=]
[1200,xfx,-->]
[400,yfx,\\]
[400,yfx,rem]
